src/BehaviorsSDKManaged/ManagedUnitTests/ManagedUnitTests.csproj

```diff
@@ -89,11 +83,14 @@
     <Prefer32Bit>true</Prefer32Bit>
     <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
   </PropertyGroup>
+  <PropertyGroup>
+	<OutputPath>..\..\..\out\$(SolutionName)\bin\$(Platform)\$(Configuration)\</OutputPath>
```

Comment:
>	 [](start = 0, length = 1)

nit: tab -> spaces

Reply:
I went through the csproj files and cleaned up the tabs for both the native and managed projects.